azurerm/resource_arm_storage_account.go

```diff
@@ -223,6 +223,33 @@ func resourceArmStorageAccount() *schema.Resource {
 				Sensitive: true,
 			},
 
+			"identity": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
```

Comment:
As there is only one possible value for now should we just default it to `SystemAssigned`?